package.json

```diff
@@ -17,9 +17,8 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "next": "*",
-    "react": "*",
-    "react-dom": "*"
+    "react": "^16.8 || ^17 || ^18",
+    "react-dom": "^16.8 || ^17 || ^18"
```

Comment:
hooks were introduced as a feature in react 16.8, so specifying peerDeps as `*` is a mistake!